meta/meta.go

```diff
@@ -59,25 +59,26 @@ var (
 //
 
 var (
-	mMetaPrefix       = []byte("m")
-	mNextGlobalIDKey  = []byte("NextGlobalID")
-	mSchemaVersionKey = []byte("SchemaVersionKey")
-	mDBs              = []byte("DBs")
-	mDBPrefix         = "DB"
-	mTablePrefix      = "Table"
-	mSequencePrefix   = "SID"
-	mSeqCyclePrefix   = "SequenceCycle"
-	mTableIDPrefix    = "TID"
-	mIncIDPrefix      = "IID"
-	mRandomIDPrefix   = "TARID"
-	mBootstrapKey     = []byte("BootstrapKey")
-	mSchemaDiffPrefix = "Diff"
-	mPolicies         = []byte("Policies")
-	mPolicyPrefix     = "Policy"
-	mPolicyGlobalID   = []byte("PolicyGlobalID")
-	mPolicyMagicByte  = CurrentMagicByteVer
-	mDDLTableVersion  = []byte("DDLTableVersion")
-	mConcurrentDDL    = []byte("concurrentDDL")
+	mMetaPrefix         = []byte("m")
+	mNextGlobalIDKey    = []byte("NextGlobalID")
+	mSchemaVersionKey   = []byte("SchemaVersionKey")
+	mDBs                = []byte("DBs")
+	mDBPrefix           = "DB"
+	mTablePrefix        = "Table"
+	mSequencePrefix     = "SID"
+	mSeqCyclePrefix     = "SequenceCycle"
+	mTableIDPrefix      = "TID"
+	mIncIDPrefix        = "IID"
+	mRandomIDPrefix     = "TARID"
+	mBootstrapKey       = []byte("BootstrapKey")
+	mSchemaDiffPrefix   = "Diff"
+	mPolicies           = []byte("Policies")
+	mPolicyPrefix       = "Policy"
+	mPolicyGlobalID     = []byte("PolicyGlobalID")
+	mPolicyMagicByte    = CurrentMagicByteVer
+	mDDLTableVersion    = []byte("DDLTableVersion")
+	mConcurrentDDL      = []byte("concurrentDDL")
+	mInFlashbackCluster = []byte("InFlashbackCluster")
```

Comment:
IMHO it is not an elegant design to introduce `mInFlashbackCluster`, but it is straightforward and easy for quick-and-safe implementation.

I can accept it for now.